tiled/server/app.py

```diff
@@ -858,6 +875,12 @@ async def capture_metrics_prometheus(request: Request, call_next):
                 metrics.capture_request_metrics(request, response)
             return response
```

Comment:
Do we want to remove `except Exception` here since we are now registering exception middleware above?

Reply:
I tried that. The `test_metrics.py` unit tests fail; the metrics never see the 500 responses.

In [4b5aaa9](https://github.com/bluesky/tiled/pull/673/commits/4b5aaa9947d926488650ebec19da1e7fd5630bbd) I explain in a comment what I think is going on here.

Reply:
oooh, I see...the respsonse is there to satisfy the `finally` block that still logs the metric. Cool. Yeah that comment helps. Thanks.